pybandits/cmab.py

```diff
@@ -208,11 +208,19 @@ class CmabBernoulli(BaseCmabBernoulli):
         bandit strategy.
     """
 
-    actions: Dict[ActionId, BayesianLogisticRegression]
+    actions: Dict[ActionId, BayesianNeuralNetwork]
     strategy: ClassicBandit
     predict_with_proba: bool = False
     predict_actions_randomly: bool = False
 
+    def __eq__(self, other: "CmabBernoulli") -> bool:
```

Comment:
```suggestion
    def __eq__(self, other) -> bool:
          if type(self) is not type(other):
            return False
```

You can also return NotImplemented if you prefer, to let the interpreter know that equality is not defined and allow the other object's __eq__ method (if it exists) to handle the comparison.

Reply:
Thank you Alessandro for the feedback.